kernel/src/error.rs

```diff
@@ -171,7 +171,7 @@ impl Error {
         Self::FileNotFound(path.to_string())
     }
     pub fn missing_column(name: impl ToString) -> Self {
-        Self::MissingColumn(name.to_string())
+        Self::MissingColumn(name.to_string()).with_backtrace()
```

Comment:
temporary debugging aid? or intentional?

Reply:
intentional, since this happens in multiple places it's useful to have a backtrace.